### ISSUE TITLE
Warning logs fired multiple times for a single timestamp

### ISSUE THREAD

**Issue by pierresegonne** [electricitymaps/electricitymaps-contrib#3648]
## Description

When looking at `https://storage.googleapis.com/electricitymap-parser-logs/US-CAL-CISO.html`, we can see that the warning logs for clamping solar production to 0 are fired multiple times for a same event. This makes the log file much more difficult to read. 

## Task List

- [ ] Fix firing of the log
- [ ] Clean up log file

**Comment by skovhus** [comment]
I'm pretty sure this is for different datapoints/events. Also see that we have multiple `invalid point` logs.

The `timestamp` in the log files are the timestamp that the server logged the line with a second resolution, so naturally multiple timestamps will contain multiple log lines.

**Comment by pierresegonne** [comment]
Yes indeed, you can see for the EIA parser that adding the series that resulted in the error would help distinguish

**Comment by VIKTORVAV99** [comment]
@pierresegonne is this still a problem or can we close this?

**Comment by pierresegonne** [comment]
I've not looked since

**Comment by VIKTORVAV99** [comment]
Closing this, feel free to re-open if it's still a issue.
